package.json

```diff
@@ -26,7 +26,7 @@
     "jest-each": "^26.6.2",
     "jest-expect-message": "^1.0.2",
     "jest-junit": "^12.0.0",
-    "lerna": "^3.20.2",
+    "lerna": "3.22.1",
```

Comment:
maybe we can get rid of `lerna`, as we are not using it at all, it seems to me that yarn or npm workspaces are trendier to manage monorepos now.

Reply:
yes, we'll consider this in the near future